src/sagemaker/pytorch/defaults.py

```diff
@@ -12,7 +12,7 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
-PYTORCH_VERSION = '0.4'
+PYTORCH_VERSION = '1.1.0'
```

Comment:
note the comment for this variable. We stopped updating it because doing so could break existing workflows.

Reply:
Thanks for calling this out! I've reverted it to 0.4

Reply:
but it's been changed back to 1.1.0 again?

Reply:
My bad, the branch got multiple updates over the time, and I got missed this when I resolve conflicts. Thanks for pointing this out.